ui/page/governance/agenda_vote_modal.go

```diff
@@ -0,0 +1,307 @@
+package governance
+
+import (
+	"sort"
+
+	"gioui.org/font/gofont"
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/widget"
+	"gioui.org/widget/material"
+
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type agendaVoteModal struct {
+	*load.Load
+	modal decredmaterial.Modal
+
+	LiveTickets []*dcrlibwallet.Transaction
+
+	agenda    *dcrlibwallet.Agenda
+	isVoting  bool
+	loadCount int
+
+	consensusPage *ConsensusPage
```

Comment:
Do not link to a page, use a callback instead to execute a method on the calling page whenever necessary here. Compare a callback is used in the create password modal. https://github.com/planetdecred/godcr/blob/c66d74aaee17c798bc64b976c61bc43edb7eae67/ui/modal/create_password_modal.go#L136

Reply:
https://github.com/planetdecred/godcr/pull/729/files#diff-c857bf384d3c25f21adf5db9d32b10a475d3fcee66adaa7d3620eb61518227bbR253, the page liking was used to refresh the agendas list. Currently not seeing the feasibility on linking the consensus page and agenda vote modal using a callback

can you push a commit to this effect